modules/ipi-install-configuring-storage-on-nodes.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-storage-on-nodes_{context}"]
+= Optional: Configuring storage on nodes
+
+You can make changes to operating systems on {product-title} nodes by creating `MachineConfig` objects that are managed by the Machine Config Operator (MCO).
+
+The `MachineConfig` specification includes an ignition config for configuring the machines at first boot. This config object can be used to modify files, systemd services, and other operating system features running on on {product-title} machines.
+
+.Procedure
+
+Use the ignition config to configure storage on the nodes. The following `MachineSet` manifest example demonstrates how to add a partition to the sda device on primary nodes. In this example, apply the manifest before installation to have a partition named `recovery` with a size of 16 GiB on the primary nodes.
+
+. Create the new `MachineConfig` manifest as shown in the following example:
++
+[source,terminal]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: primary
+  name: 10_primary_storage_config
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+        - device: </dev/xxyN>
```

Comment:
in line 16 you mention sda device but in manifest I see xxyN
please consider to remove sda or add /dev/da here

Reply:
Updated content to use the naming listed in [Partition Naming Scheme](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/managing_storage_devices/index#partition-naming-scheme_disk-partitions).